src/main/java/Task.java

```diff
@@ -0,0 +1,31 @@
+public class Task {
+    private static int taskCounter = 0;
```

Comment:
```suggestion
    private static int numTasks = 0;
```
Do you think this is a clearer name?

Reply:
I think numberOfTasks would be better but a little long so I think I'll keep this thanks!